packages/tinacms/src/admin/pages/CollectionCreatePage.tsx

```diff
@@ -192,11 +199,19 @@ export const RenderForm = ({
       extraSubscribeValues: { active: true, submitting: true, touched: true },
       onChange: (values) => {
         if (!values?.submitting) {
-          // keeps the forms relative path in sync with the filename
-          form.relativePath =
-            schemaCollection.path +
-            folderName +
-            `/${values?.values?.filename}.${schemaCollection.format || 'md'}`
+          const fileName: string = values?.values?.filename
+          // If the filename starts with "/" then it is an absolute path and we should not append the folder name
+          if (fileName?.startsWith('/')) {
+            form.relativePath =
+              schemaCollection.path +
+              `${values?.values?.filename}.${schemaCollection.format || 'md'}`
+          } else {
+            // keeps the forms relative path in sync with the filename
+            form.relativePath =
+              schemaCollection.path +
+              folderName +
+              `/${values?.values?.filename}.${schemaCollection.format || 'md'}`
+          }
```

Comment:
```suggestion
    form.relativePath =
      schemaCollection.path +
      appendFolder +
      `${values?.values?.filename}.${schemaCollection.format || 'md'}`
```
^ I think you can remove the conditional like so

Reply:
Yup this makes more sense to me